src/igvfd/tests/test_schema_modification.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+
+def test_patch_cas_species(modification, testapp):
+    res = testapp.patch_json(modification['@id'],
+                             {'cas_species': 'Streptococcus pyogenes (Sp)'})
+    assert res.status_code == 200
+    res = testapp.patch_json(modification['@id'],
+                             {'cas_species': ''}, expect_errors=True)
+    assert res.status_code == 422
+
+
+def test_read_cas_species(modification, testapp):
+    res = testapp.get(modification['@id'])
+    assert res.json['cas_species'] == 'Streptococcus pyogenes (Sp)'''
```

Comment:
i am not sure what is this testing? and it also ends with '''?

Reply:
I wanted to test the default assignment, but i realize that the fixture has `cas_species` explicitly assigned, and the test doesn't pass unless it is explicitly assigned.  I removed the ''' at the end.

Reply:
will remove now that i've removed default